.github/CODEOWNERS

```diff
@@ -6,27 +6,20 @@
 ################
 # As of 1/26/2023 these paths have no owners:
 
-# /
+# /**
 # /.devcontainer/
 # /.vscode/
 # /conda-releaselogs/
 # /doc/
 # /scripts/
 # /tools/
 
-###########
-# Eng Sys
-###########
-/eng/                                                                @scbedd @weshaggard @benbp
-/**/tests.yml                                                        @scbedd @benbp
-/**/ci.yml                                                           @scbedd @benbp
-
 ################
 # Automation
 ################
 
 # Git Hub integration and bot rules
-/.github/                                       @jsquire @ronniegeraghty
+/.github/                                                            @jsquire @ronniegeraghty
```

Comment:
@jsquire I assume once we remove the fabric bot configuration you are no longer going to be interested in reviews for things under this directory correct? Or are you still interested in Codeowner changes or something else?

Reply:
Correct.   Once we're no longer trying to mirror CODEOWNERS changes into rules, I will happily remove myself from all of these other than .NET.